qa/automation/ScoringPortal.Tests.Scenarios/signin_with_google_tests.feature

```diff
@@ -0,0 +1,84 @@
+
+
+Feature: sign-in via google account
+
+    """
+    since we're going to check only the functionality of application to 
+    sign-in with google the last one itself shouldn't be checked 
+    """
+
+    Background:
+        Given valid email, password for google account without 2-step verification
+        # in order to run .py test 
+        And the vpn is used
+        And we already have "https://qa.ddso-spot.quantori.com/" aplication
+        # And the field 'email' is empty
+        # And the field 'password' is empty
+
+
+    Scenario: sign-in successfully
+        Given  a set of valid data
+        # let's pretend login 'user1@quantori.academy' with the password
+        # 12345678 is a valid data since i don't want to show mine :)
+        |email                  | password |
+        | user@quantori.academy | 12345678 |
+        
+        And sign in google account
+        And enter email into "email" field
+        And click <next> button
+        And type the password into the "password" field
+        And click <next> button
+        And use_openvnp_config is True
```

Comment:
I thought it is already used in the background

Reply:
i meant an explicit assertion error here or even flag when running test